pages/system/tabs.js

```diff
@@ -60,37 +76,49 @@ const TAB_GROUP_THREE = [
   { value: '2', label: 'Aquarius' },
   { value: '3', label: 'Pisces' },
 ];`}
-        </System.CodeBlock>
-        <br />
-        <System.P>Define the Tab value states and handle the state when a tab is changed.</System.P>
-        <br />
-        <System.CodeBlock>
-          {`state = {
-  eight: '1',
-  nine: '1',
-  ten: '1',
-};
-
-_handleChange = (e) => {
-  this.setState({ [e.target.name]: e.target.value });
-};`}
         </System.CodeBlock>
         <br />
         <System.P>Declare the TabGroup component.</System.P>
         <br />
         <System.CodeBlock>
-          {`<System.TabGroup 
-  name="eight"
-  options={TAB_GROUP_TWO}
-  value={this.state.eight}
-  onChange={this._handleChange} />
+          {`class ExampleOne extends React.Component {
+   state = { exampleOne: '1' }
+
+   _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`